HealthyTests/Classes/Utilities/Validators/EmailValidatorsTests.swift

```diff
@@ -0,0 +1,46 @@
+import XCTest
+@testable import Healthy
+final class EmailValidatorsTests: XCTestCase {
+    // MARK: - Properties
```

Comment:
```suggestion

    // MARK: - Properties
    
```
Follow that pattern in all of your MARKs.